addons/l10n_gr_edi/data/template.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="send_invoice">
+        <InvoicesDoc xmlns="http://www.aade.gr/myDATA/invoice/v1.0"
+                     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                     xmlns:icls="https://www.aade.gr/myDATA/incomeClassificaton/v1.0"
+                     xmlns:ecls="https://www.aade.gr/myDATA/expensesClassificaton/v1.0"
+                     xsi:schemaLocation="http://www.aade.gr/myDATA/invoice/v1.0/InvoicesDoc-v0.6.xsd">
+            <invoice t-foreach="invoices" t-as="invoice">
+                <issuer>
+                    <vatNumber t-out="invoice.get('issuer_vat')"/>
+                    <country t-out="invoice.get('issuer_country')"/>
+                    <branch t-out="invoice.get('issuer_branch')"/>
+                    <name t-out="invoice.get('issuer_name')"/>
+                    <address>
+                        <postalCode t-out="invoice.get('issuer_postal_code')"/>
+                        <city t-out="invoice.get('issuer_city')"/>
+                    </address>
+                </issuer>
+                <counterpart>
+                    <vatNumber t-out="invoice.get('counterpart_vat')"/>
+                    <country t-out="invoice.get('counterpart_country')"/>
+                    <branch t-out="invoice.get('counterpart_branch')"/>
+                    <name t-out="invoice.get('counterpart_name')"/>
+                    <address>
+                        <postalCode t-out="invoice.get('counterpart_postal_code')"/>
+                        <city t-out="invoice.get('counterpart_city')"/>
+                    </address>
+                </counterpart>
+                <invoiceHeader>
+                    <series t-out="invoice.get('header_series')"/>
+                    <aa t-out="invoice.get('header_aa')"/>
+                    <issueDate t-out="invoice.get('header_issue_date')"/>
+                    <invoiceType t-out="invoice.get('header_invoice_type')"/>
+                    <currency t-out="invoice.get('header_currency')"/>
+                    <correlatedInvoices t-out="invoice.get('header_correlate')"/>
+                </invoiceHeader>
+                <paymentMethods>
+                    <paymentMethodDetails t-foreach="invoice.get('payment_details')" t-as="payment_detail">
```

Comment:
```suggestion
                    <paymentMethodDetails t-foreach="invoice.get('payment_details', [])" t-as="payment_detail">
```
Otherwise :boom: